tests/testthat/test-derive_params_growth_age.R

```diff
@@ -0,0 +1,40 @@
+# derive_params_growth_age ----
+
+## Test 1: derive_params_growth_age works ----
+test_that("derive_params_growth_age Test 1: derive_params_growth_age works", {
+  vs_data <- tibble::tribble(
+    ~USUBJID, ~SEX, ~AGECUR, ~AGEU, ~VSTESTCD, ~VSSTRESN,
+    "1001", "M", 30, "days", "WEIGHT", 10,
+    "1002", "F", 25, "months", "WEIGHT", 100,
+  )
+
+  fake_meta <- tibble::tribble(
```

Comment:
to avoid using dependencies in data I created an entirely fake metadata example that as long as the structure is correct will make the appropriate calcuation